earth2studio/data/arco.py

```diff
@@ -125,6 +130,15 @@ async def _async_init(self) -> None:
         self.level_coords = await (await self.zarr_group.get("level")).getitem(
             slice(None)
         )
+        # Model-level store
+        ml_zstore = zarr.storage.FsspecStore(
+            fs,
+            path="/gcp-public-data-arco-era5/ar/model-level-1h-0p25deg.zarr-v1",
+        )
+        self.ml_zarr_group = await zarr.api.asynchronous.open(store=ml_zstore, mode="r")
+        self.ml_level_coords = await (await self.ml_zarr_group.get("hybrid")).getitem(
```

Comment:
Am I missing something? Where is this used?

Reply:
Oops, nice catch. I think some things got removed by accident by me during cleanup. Corrected this method so now the model levels file should actually be getting used (confirmed with checks logging the zarr group used for each request)